app/views/idv/in_person/state_id.html.erb

```diff
@@ -1,13 +1,78 @@
 <% title t('titles.doc_auth.verify') %>
 
-<h1 class="h3 margin-y-0">
-  <%= t('in_person_proofing.headings.state_id') %>
-</h1>
+
+<%= render PageHeadingComponent.new.with_content(t('in_person_proofing.headings.state_id')) %>
+
+<p>
+  <%= t('in_person_proofing.body.state_id.info_html') %>
+</p>
 
 <%= validated_form_for :doc_auth,
                        url: url_for,
                        method: 'put',
                        html: { autocomplete: 'off', class: 'margin-y-5' } do |f| %>
+
+  <div class="margin-bottom-4">
+    <%= render ValidatedFieldComponent.new(
+          name: :first_name,
+          form: f,
+          label: t('in_person_proofing.form.state_id.first_name'),
+          label_html: { class: 'usa-label' },
```

Comment:
@aduth I added this class to all the fields on this page to tighten up spacing between label and hint; here's before & after:

![usa-label-before-after](https://user-images.githubusercontent.com/45415133/173661156-068f8c1d-97df-4ff0-9a83-874bfd82068e.gif)

The resulting labels have both `text-bold` and `usa-label` styles (`text-bold` is set as the default [in SimpleForm config](https://github.com/18F/identity-idp/blob/main/config/initializers/simple_form.rb#L29)). Is this an acceptable way to get the result I want?

Reply:
I think that's okay, yeah. We're kind of inconsistent with it, but we've done this same pattern in [some places](https://github.com/18F/identity-idp/blob/497887e897525d7872f99d87f0cf9f49a65c62a5/app/components/phone_input_component.html.erb#L21) (and not [others](https://github.com/18F/identity-idp/blob/497887e897525d7872f99d87f0cf9f49a65c62a5/app/views/shared/_ssn_field.html.erb#L18)). It ought to be standardized to use the design system label across the board, but we're not quite there yet (maybe part of LG-6097).